hack/common.sh

```diff
@@ -213,6 +211,7 @@ os::build::internal::build_binaries() {
       fi
 
       local platform_gotags_envvar=OS_GOFLAGS_TAGS_$(echo ${platform} | tr '[:lower:]/' '[:upper:]_')
+      local platform_gotags_test_envvar=OS_GOFLAGS_TAGS_TEST_$(echo ${platform} | tr '[:lower:]/' '[:upper:]_')
```

Comment:
@liggitt is the main potential impact with gssapi

In general test tags should be explicit - if we need gssapi in integration, we need to call that out (which means it would fail on Macs, which it doesn't).